tests/test_transaction.py

```diff
@@ -7,6 +7,11 @@
 from psycopg import Connection, ProgrammingError, Rollback
 
 
+@pytest.fixture
+def conn(conn, pipeline):
+    return conn
```

Comment:
This effectively duplicates almost all tests in the module. That a simple and exhaustive approach, but perhaps it's too much from developer point of view? If so, we can instead just pull the `pipeline` fixture in test cases where it matters (which is to be determined).

Reply:
It seems a good idea for me. If large part of the tests make sense to be in both versions we can keep it this way. If let's say half of them or more don't make sense then maybe we can introduce the pipeline fixture where we need.

If we have effectively duplicated some of the tests in the test_pipeline files, we can drop them from there.